Files/Scripts/Options/The Sealed Palace.psm1

```diff
@@ -163,7 +194,8 @@ function CreateTabMain() {
     CreateReduxCheckBox -Name "InstantClaimCheck" -Text "Instant Claim Check"         -Info "Allows you to use the claim check immediately to get the biggoron's sword"                                               -Credits "Randomizer & Anthrogi (ported)"
     CreateReduxCheckBox -Name "BlackBars"         -Text "No Black Bars (Z-Targeting)" -Info "Removes the black bars shown on the top & bottom of the screen during Z-targeting"                                       -Credits "Admentus & Anthrogi (ported)"
     CreateReduxCheckBox -Name "TextSpeed"         -Text "2x Text Speed"               -Info "Makes text go 2x as fast"                                                                                                -Credits "Admentus & Anthrogi (ported)"
-    CreateReduxCheckBox -Name "FastArrows"        -Text "Less Magic Arrows Cooldown"  -Info "The burst animation for Fire, Ice and Light Arrows are shorter`nAllows Link to shoot the next magic arrow a bit quicker" -Credits "Anthrogi"
```

Comment:
You do not need to state you ported it, the code changes were yours to begin in the first place. Only do that when it isn't yours to begin with, then you ported it over.

Reply:
I see although for me, I used the indication "ported" because even if I found the function, it's a copy over from the originally intended baserom I supported it with which was ntsc 1.0 for both OoT & MM, so I felt the need to but I'll let that go then.